### ISSUE TITLE
Better support for Link-only actions

### ISSUE THREAD

**Issue by nickoneill** [5calls/5calls#353]
We'd like to promote some actions that are just links, i.e. sending a comment to the FCC via their own form URL. But we still want to track these results so we can know how many people have given feedback.

Currently the web client supports Link-only issues, but it needs to be modified to have a result button that says "I sent this feedback" or something.

We can include this in our custom outcomes from the server if that makes sense, or it can be hardcoded into the Link component if it would work better that way.

(also, what should happen after the Result is entered? Done page?)

**Comment by kamoh** [comment]
Can the result button be something as generic as `I did it`? And when it's done, can we use the current `Great work` page that you see when you contact all the people for a given issue?

**Comment by nickoneill** [comment]
Yep, simple copy on the button that indicates that the action was completed is fine. Should follow the same rules as the case where you're done with your set of contacts after clicked.

**Comment by nickoneill** [comment]
Some work on the "I did it" button support for teams was done here: https://github.com/5calls/5calls/commit/c15342ff0d936179da667f5e139af7847b96b65e
